lib/components/Toggle/Toggle.tsx

```diff
@@ -34,6 +38,12 @@ export interface ToggleProps extends React.Props<ToggleType> {
     attr?: ToggleAttributes;
 }
 
+const StyledToggleOnButton = styled(Attr.button)`
+    &&&&&& {
```

Comment:
What's this magic - some places have 4 while this one has 6?

Reply:
It's the syntax in styled-component to bump up the style priority to override default one.